README.md

```diff
@@ -70,21 +70,31 @@ More details are available [in the book][reference-link-implementation-details].
 
 ### Examples
 
-Currently we provide two basic examples of running AlephBFT. The first one: `ordering` implements a committee member that is not
-cryptographically secure and serves only as a working example of what traits need to be implemented and how to implement them.
-For example, you may run the following command
+We provide two basic examples of running AlephBFT, both of which are not cryptographically secure, and assume honest, but possibly malfunctioning, participants.
+
+The first one, `crash_recovery`, implements a simple node that produces data items, and then waits for them to be finalized. It can also perform a simulated crash after creating a specified number of items.
+
+For example, you may run the following command:
 ```
-cd ./examples/ordering
-./run.sh 5
+cd ./examples/crash_recovery
+./run.sh --n-working-nodes 7 --n-nodes-to-crash 3 --n-crashes 4 --n-ordered-per-crash 20 --restart-delay 5
```

Comment:
While it would make sense to have randomized tests, I agree with @lesniak43 that maybe the current version is OK for a basic example. Let's wrap it up and perhaps work on more sophisticated scenarios later in the future.